greynoise/greynoise.yaml.example

```
@@ -0,0 +1,3 @@
+---
+
+gn_api_key: "{{ st2kv.system.gn_api_key }}"
```

This is an example config value, and it will only be explicitly set if it is overriding the `gn_api_key` key from the key-value store, so setting it explicitly to pull from that in the example gives your users the wrong impression.

I would make this something that is obviously fake, like:

```suggestion
gn_api_key: GREYNOISE_SECRET_API_KEY
```